pkg/upgrade/upgrade.go

```diff
@@ -615,3 +615,179 @@ func cleanupModelControllerLegacyDeployment(ctx context.Context, cli client.Clie
 
 	return nil
 }
+
+// TODO: to be removed: https://issues.redhat.com/browse/RHOAIENG-21080
+func PatchOdhDashboardConfig(ctx context.Context, cli client.Client) error {
```

Comment:
Probably this function should also check the current and previous version to determine if the patch should be applied ? something like https://github.com/opendatahub-io/opendatahub-operator/blob/main/pkg/upgrade/upgrade.go#L503

Reply:
yep it can be done, although I may need some guidance on which versions should be patched
@Gkrumbach07 ^

Reply:
I don’t think we need one specifically. The earliest probably would be coming from 2.8

Reply:
@lburgazzoli , added logic to check if currentVersion>previousVersion and only then patch.